examples/run_Hahn_echo.py

```diff
@@ -30,128 +40,86 @@
 config_dict["echo_counter"] += 1
 filename = f"{config_dict['date']}_{config_dict['chemical']}_{config_dict['type']}"
 # }}}
-# {{{set phase cycling
-phase_cycling = True
-if phase_cycling:
-    ph1_cyc = r_[0, 1, 2, 3]
-    nPhaseSteps = 4
-if not phase_cycling:
-    ph1_cyc = 0.0
-    nPhaseSteps = 1
-# }}}
```

Comment:
the default for run_spin_echo is to use this phase cycle so I don't see a point in keeping this